### DISCUSSION TITLE
SelectAction in ListPage header

### DISCUSSION THREAD

**Discussion by bvicini** [filamentphp/filament#13834] [Help]
### Package

Panel builder

### Package Version

v.3.2.96

### How can we help you?

I'm trying to use SelectAction to display a Select in the header of ListPage or in the header of the table itself in order to update the page query based on the selected option (i want to implement a mandatory filter). The select appears but when i change the option the action() is not triggered. Is SelectAction the right way to achieve this goal? What am I doing wrong? Ty

**Comment by rusowyler** [comment]
Hey @bvicini same question here. I want to change the table query based on a header dropdown value. 
Were you able to solve it? Any idea?

**Reply by bvicini** [reply]
Heelo. No, I just leave that problem alone hoping to find some help in the meantime. Sory

**Comment by bvicini** [comment]
I just found that the answer to Wire:change Event in SelectAction #8741 is good for this case. Ty
